### PR TITLE
WRN-12943: Improve window size changes

### PR DESCRIPTION
**Pull request by adrian-cocoara-lgp**
Added some media queries so the app does not break its layout on smaller resolution

### CODE REVIEW

File: sandstone/custom-skin/src/components/TripleField/TripleField.module.less [hunk 1]
```diff
@@ -7,10 +7,83 @@
...
+        .greenInput,
+        .blueInput {
+            height: 1.5rem;
+            max-width: 2.7rem;
+            margin: 0.1rem;
+            &.inputField {
+                .input {
+                    font-size: 0.6rem;
+                }
+            }
```
Comment (daniel-stoian-lgp) [inline sandstone/custom-skin/src/components/TripleField/TripleField.module.less:37]: I believe we can use just .input, in this file and in SingleField. What do you think?

File: sandstone/custom-skin/src/components/TripleField/TripleField.module.less [hunk 2]
```diff
@@ -7,10 +7,83 @@
...
+    .tripleField {
+        flex-direction: column;
+        align-items: flex-start;
+        .tripleText {
+            font-size: 0.6rem;
+        }
+        .redInput,
+        .greenInput,
+        .blueInput {
+            height: 1.5rem;
```
Comment (daniel-stoian-lgp) [inline sandstone/custom-skin/src/components/TripleField/TripleField.module.less:30]: I think you also need to adjust the height of child elements: Example for .inputHighlight. The same for .input
image

Please check everywhere you set the height on .inputField

File: sandstone/custom-skin/src/views/MainPanel.module.less [hunk 3]
```diff
@@ -61,3 +61,166 @@
...
+            flex-direction: column;
+            align-items: center;
+            & > * {
+                min-width: 380px;
+            }
+        }
+    }
+}
+
+@media screen and (min-width: 0px) and (max-width: 1370px) {
```
Comment (daniel-stoian-lgp) [inline sandstone/custom-skin/src/views/MainPanel.module.less:172]: is the min-width necessary?

File: sandstone/custom-skin/src/views/MainPanel.module.less [hunk 4]
```diff
@@ -61,3 +61,166 @@
...
+            }
+            .previewRangePicker {
+                margin: 0.2rem 2rem;
+            }
+            .previewDropdown {
+                margin: 0 2rem;
+            }
+            .previewPopup {
+                margin: 0.7rem 2rem;
+            }
```
Comment (daniel-stoian-lgp) [inline sandstone/custom-skin/src/views/MainPanel.module.less:223]: can you group common styles?

File: sandstone/custom-skin/src/views/MainPanel.module.less [hunk 5]
```diff
@@ -61,3 +61,166 @@
...
+            margin: 0 4rem;
+        }
+        .previewSwitchItem {
+            margin: 0 4rem;
+        }
+        .previewSlider {
+            margin: 0.2rem 4rem;
+        }
+        .previewRangePicker {
+            margin: 0.3rem 4rem;
```
Comment (daniel-stoian-lgp) [inline sandstone/custom-skin/src/views/MainPanel.module.less:155]: can you group common styles?

File: sandstone/custom-skin/src/views/MainPanel.module.less [hunk 6]
```diff
@@ -61,3 +61,166 @@
...
+        .previewComponents {
+            .previewCheckboxItem {
+                margin: 0 4.5rem;
+            }
+            .previewSwitchItem {
+                margin: 0 4.5rem;
+            }
+            .previewSlider {
+                margin: 0 4.5rem;
+            }
```
Comment (daniel-stoian-lgp) [inline sandstone/custom-skin/src/views/MainPanel.module.less:132]: can you group common styles?

File: sandstone/custom-skin/src/views/MainPanel.module.less [hunk 7]
```diff
@@ -61,3 +61,166 @@
...
+                margin: 0 2.3rem;
+            }
+            .previewSlider {
+                margin: 0 2.7rem;
+            }
+            .previewRangePicker {
+                margin: 0 1rem;
+            }
+            .previewPopup {
+                min-width: 5rem;
```
Comment (daniel-stoian-lgp) [inline sandstone/custom-skin/src/views/MainPanel.module.less:89]: can you group common styles?

File: sandstone/custom-skin/src/views/MainPanel.module.less (hunk 3 above)
Comment (adrian-cocoara-lgp) [reply sandstone/custom-skin/src/views/MainPanel.module.less:172]: no, it's not. I removed it

File: sandstone/custom-skin/src/components/TripleField/TripleField.module.less (hunk 1 above)
Comment (adrian-cocoara-lgp) [reply sandstone/custom-skin/src/components/TripleField/TripleField.module.less:37]: Agreed

**Review by vladut-clapou-lgp — APPROVED** [review]
LGTM

File: sandstone/custom-skin/src/views/MainPanel.module.less [hunk 8]
```diff
@@ -1,63 +1,214 @@
...
+			font-size: 42px;
+		}
+	}
+
+	.previewSection {
+		.previewComponents {
+			* {
+				font-size: 42px;
+			}
+			.previewButtons {
```
Comment (stanca-pop-lgp) [inline sandstone/custom-skin/src/views/MainPanel.module.less:191]: Please leave an empty space above.

File: sandstone/custom-skin/src/views/MainPanel.module.less [hunk 9]
```diff
@@ -1,63 +1,214 @@
...
+				display: flex;
+				justify-content: center;
+				& > * {
```
Comment (stanca-pop-lgp) [inline sandstone/custom-skin/src/views/MainPanel.module.less:194]: Please leave an empty space above.

File: sandstone/custom-skin/src/views/MainPanel.module.less [hunk 10]
```diff
@@ -1,63 +1,214 @@
...
+					min-width: 380px;
+				}
+			}
+			.previewCheckboxItem,
```
Comment (stanca-pop-lgp) [inline sandstone/custom-skin/src/views/MainPanel.module.less:198]: Please leave an empty space above.

File: sandstone/custom-skin/src/views/MainPanel.module.less [hunk 11]
```diff
@@ -1,63 +1,214 @@
...
+			.previewSwitchItem,
+			.previewDropdown {
+				margin: 0 96px;
+			}
+			.previewSlider {
```
Comment (stanca-pop-lgp) [inline sandstone/custom-skin/src/views/MainPanel.module.less:203]: Please leave an empty space above.

File: sandstone/custom-skin/src/views/MainPanel.module.less [hunk 12]
```diff
@@ -1,63 +1,214 @@
...
+				margin: 6px 96px;
+			}
+			.previewRangePicker {
```
Comment (stanca-pop-lgp) [inline sandstone/custom-skin/src/views/MainPanel.module.less:206]: Please leave an empty space above.

File: sandstone/custom-skin/src/views/MainPanel.module.less [hunk 13]
```diff
@@ -1,63 +1,214 @@
...
+				margin: 9px 96px;
+			}
+			.previewPopup {
```
Comment (stanca-pop-lgp) [inline sandstone/custom-skin/src/views/MainPanel.module.less:209]: Please leave an empty space above.

File: sandstone/custom-skin/src/views/MainPanel.module.less [hunk 14]
```diff
@@ -1,63 +1,214 @@
...
+		}
+	}
+}
+
+@media screen and (min-width: 1170px) and (max-width: 1670px) {
+	.customizeSection {
+		.appTitle {
+			font-size: 51px;
+		}
+		.switchLabel {
```
Comment (stanca-pop-lgp) [inline sandstone/custom-skin/src/views/MainPanel.module.less:181]: Please leave an empty space above.

File: sandstone/custom-skin/src/views/MainPanel.module.less [hunk 15]
```diff
@@ -1,63 +1,214 @@
...
+		}
+	}
+}
+
+@media screen and (min-width: 719px) and (max-width: 1290px) {
+	.previewSection {
+		.previewButtons {
+			flex-direction: column;
+			align-items: center;
+			& > * {
```
Comment (stanca-pop-lgp) [inline sandstone/custom-skin/src/views/MainPanel.module.less:157]: Please leave an empty space above.

File: sandstone/custom-skin/src/views/MainPanel.module.less [hunk 16]
```diff
@@ -1,63 +1,214 @@
...
+			font-size: 33px;
+		}
+		.previewCheckboxItem,
+		.previewSwitchItem {
+			margin: 0 192px;
+		}
+		.previewSlider {
+			margin: 9px 192px;
+		}
+		.previewRangePicker {
```
Comment (stanca-pop-lgp) [inline sandstone/custom-skin/src/views/MainPanel.module.less:146]: Please leave an empty space above.

File: sandstone/custom-skin/src/views/MainPanel.module.less [hunk 17]
```diff
@@ -1,63 +1,214 @@
...
+@media screen and (min-width: 720px) and (max-width: 1170px) {
+	.previewSection {
+		* {
+			font-size: 33px;
+		}
+		.previewCheckboxItem,
+		.previewSwitchItem {
+			margin: 0 192px;
+		}
+		.previewSlider {
```
Comment (stanca-pop-lgp) [inline sandstone/custom-skin/src/views/MainPanel.module.less:143]: Please leave an empty space above.

File: sandstone/custom-skin/src/views/MainPanel.module.less [hunk 18]
```diff
@@ -1,63 +1,214 @@
...
+	}
+}
+
+@media screen and (min-width: 720px) and (max-width: 1170px) {
+	.previewSection {
+		* {
+			font-size: 33px;
+		}
+		.previewCheckboxItem,
+		.previewSwitchItem {
```
Comment (stanca-pop-lgp) [inline sandstone/custom-skin/src/views/MainPanel.module.less:140]: Please leave an empty space above.

File: sandstone/custom-skin/src/views/MainPanel.module.less [hunk 19]
```diff
@@ -1,63 +1,214 @@
...
+		}
+	}
+}
+
+@media screen and (min-width: 720px) and (max-width: 1170px) {
+	.previewSection {
+		* {
+			font-size: 33px;
+		}
+		.previewCheckboxItem,
```
Comment (stanca-pop-lgp) [inline sandstone/custom-skin/src/views/MainPanel.module.less:139]: Please leave an empty space above.

File: sandstone/custom-skin/src/views/MainPanel.module.less [hunk 20]
```diff
@@ -1,63 +1,214 @@
...
+		}
+	}
+	.previewSection {
+		.previewComponents {
+			.previewCheckboxItem,
+			.previewSwitchItem,
+			.previewSlider {
+				margin: 0 216px;
+			}
+			.previewPopup {
```
Comment (stanca-pop-lgp) [inline sandstone/custom-skin/src/views/MainPanel.module.less:127]: Please leave an empty space above.

File: sandstone/custom-skin/src/views/MainPanel.module.less [hunk 21]
```diff
@@ -1,63 +1,214 @@
...
+		}
+	}
+}
+
+@media screen and (min-width: 720px) and (max-width: 870px) {
+	.customizeSection {
+		.appTitle {
+			font-size: 48px;
+		}
+		.switchLabel {
```
Comment (stanca-pop-lgp) [inline sandstone/custom-skin/src/views/MainPanel.module.less:116]: Please leave an empty space above.

File: sandstone/custom-skin/src/views/MainPanel.module.less [hunk 22]
```diff
@@ -1,63 +1,214 @@
...
+		.previewComponents{
+			justify-content: space-evenly;
+			.previewButtons {
+				flex-direction: column;
+				align-items: center;
+				& > * {
+					min-width: 480px;
+				}
+			}
+			.previewSlider {
```
Comment (stanca-pop-lgp) [inline sandstone/custom-skin/src/views/MainPanel.module.less:104]: Please leave an empty space above.

File: sandstone/custom-skin/src/views/MainPanel.module.less [hunk 23]
```diff
@@ -1,63 +1,214 @@
...
+			}
+		}
+	}
+}
+
+@media screen and (min-width: 500px) and (max-width: 719px) {
+	.previewSection {
+		.previewComponents{
+			justify-content: space-evenly;
+			.previewButtons {
```
Comment (stanca-pop-lgp) [inline sandstone/custom-skin/src/views/MainPanel.module.less:97]: Please leave an empty space above.

File: sandstone/custom-skin/src/views/MainPanel.module.less [hunk 24]
```diff
@@ -1,63 +1,214 @@
...
+			.previewSwitchItem {
+				margin: 0 111px;
+			}
+			.previewSlider {
+				margin: 0 129px;
+			}
+			.previewRangePicker {
+				margin: 0 48px;
+			}
+			.previewPopup {
```
Comment (stanca-pop-lgp) [inline sandstone/custom-skin/src/views/MainPanel.module.less:86]: Please leave an empty space above.

File: sandstone/custom-skin/src/views/MainPanel.module.less [hunk 25]
```diff
@@ -1,63 +1,214 @@
...
+				}
+			}
+			.previewCheckboxItem,
+			.previewSwitchItem {
+				margin: 0 111px;
+			}
+			.previewSlider {
+				margin: 0 129px;
+			}
+			.previewRangePicker {
```
Comment (stanca-pop-lgp) [inline sandstone/custom-skin/src/views/MainPanel.module.less:83]: Please leave an empty space above.

File: sandstone/custom-skin/src/views/MainPanel.module.less [hunk 26]
```diff
@@ -1,63 +1,214 @@
...
+				& > * {
+					min-width: 288px;
+					font-size: 42px;
+				}
+			}
+			.previewCheckboxItem,
+			.previewSwitchItem {
+				margin: 0 111px;
+			}
+			.previewSlider {
```
Comment (stanca-pop-lgp) [inline sandstone/custom-skin/src/views/MainPanel.module.less:80]: Please leave an empty space above.

File: sandstone/custom-skin/src/views/MainPanel.module.less [hunk 27]
```diff
@@ -1,63 +1,214 @@
...
+		.previewComponents {
+			.previewButtons {
+				flex-direction: column;
+				align-items: center;
+				& > * {
+					min-width: 288px;
+					font-size: 42px;
+				}
+			}
+			.previewCheckboxItem,
```
Comment (stanca-pop-lgp) [inline sandstone/custom-skin/src/views/MainPanel.module.less:76]: Please leave an empty space above.

File: sandstone/custom-skin/src/views/MainPanel.module.less [hunk 28]
```diff
@@ -1,63 +1,214 @@
...
+	}
+}
+
+@media screen and (min-width: 100px) and (max-width: 500px) {
+	.previewSection {
+		.previewComponents {
+			.previewButtons {
+				flex-direction: column;
+				align-items: center;
+				& > * {
```
Comment (stanca-pop-lgp) [inline sandstone/custom-skin/src/views/MainPanel.module.less:71]: Please leave an empty space above.

File: sandstone/custom-skin/src/components/TripleField/TripleField.module.less [hunk 29]
```diff
@@ -1,16 +1,95 @@
...
+	.tripleField {
+		.tripleText {
+			font-size: 42px;
+		}
+		.redInput,
+		.greenInput,
+		.blueInput {
+			height: 96px;
+			max-width: 177px;
+				.input {
```
Comment (stanca-pop-lgp) [inline sandstone/custom-skin/src/components/TripleField/TripleField.module.less:86]: Please leave an empty space above and check out the indentation.

File: sandstone/custom-skin/src/components/TripleField/TripleField.module.less [hunk 30]
```diff
@@ -1,16 +1,95 @@
...
+					font-size: 42px;
+				}
+			.input,
```
Comment (stanca-pop-lgp) [inline sandstone/custom-skin/src/components/TripleField/TripleField.module.less:89]: Please leave an empty space above.

File: sandstone/custom-skin/src/components/TripleField/TripleField.module.less [hunk 31]
```diff
@@ -1,16 +1,95 @@
...
+		}
+	}
+}
+
+@media screen and (min-width: 1320px) and (max-width: 1670px) {
+	.tripleField {
+		.tripleText {
+			font-size: 42px;
+		}
+		.redInput,
```
Comment (stanca-pop-lgp) [inline sandstone/custom-skin/src/components/TripleField/TripleField.module.less:81]: Please leave an empty space above.

File: sandstone/custom-skin/src/components/TripleField/TripleField.module.less [hunk 32]
```diff
@@ -1,16 +1,95 @@
...
+			margin: 6px;
+				.input {
+					font-size: 42px;
+				}
+			.input,
+			.inputHighlight {
+				height: 96px;
+			}
+		}
+		.redInput,
```
Comment (stanca-pop-lgp) [inline sandstone/custom-skin/src/components/TripleField/TripleField.module.less:69]: Please leave an empty space above.

File: sandstone/custom-skin/src/components/TripleField/TripleField.module.less [hunk 33]
```diff
@@ -1,16 +1,95 @@
...
+		.greenInput,
+		.blueInput {
+			height: 96px;
+			max-width: 177px;
+			margin: 6px;
+				.input {
+					font-size: 42px;
+				}
+			.input,
```
Comment (stanca-pop-lgp) [inline sandstone/custom-skin/src/components/TripleField/TripleField.module.less:64]: Please leave an empty space above.

File: sandstone/custom-skin/src/components/TripleField/TripleField.module.less [hunk 34]
```diff
@@ -1,16 +1,95 @@
...
+		.tripleText {
+			font-size: 42px;
+		}
+		.redInput,
+		.greenInput,
+		.blueInput {
+			height: 96px;
+			max-width: 177px;
+			margin: 6px;
+				.input {
```
Comment (stanca-pop-lgp) [inline sandstone/custom-skin/src/components/TripleField/TripleField.module.less:61]: Please leave an empty space above and check indentation.

File: sandstone/custom-skin/src/components/TripleField/TripleField.module.less [hunk 35]
```diff
@@ -1,16 +1,95 @@
...
+}
+
+@media screen and (min-width: 830px) and (max-width: 1320px) {
+	.tripleField {
+		flex-direction: column;
+		align-items: flex-start;
+		.tripleText {
+			font-size: 42px;
+		}
+		.redInput,
```
Comment (stanca-pop-lgp) [inline sandstone/custom-skin/src/components/TripleField/TripleField.module.less:55]: Please leave an empty space above.

File: sandstone/custom-skin/src/components/TripleField/TripleField.module.less [hunk 36]
```diff
@@ -1,16 +1,95 @@
...
+			margin-left: 9px;
+		}
+	}
+}
+
+@media screen and (min-width: 830px) and (max-width: 1320px) {
+	.tripleField {
+		flex-direction: column;
+		align-items: flex-start;
+		.tripleText {
```
Comment (stanca-pop-lgp) [inline sandstone/custom-skin/src/components/TripleField/TripleField.module.less:52]: Please leave an empty space above.

File: sandstone/custom-skin/src/components/TripleField/TripleField.module.less [hunk 37]
```diff
@@ -1,16 +1,95 @@
...
+			margin: 6px;
+			.input {
+				font-size: 27px;
+			}
+			.input,
+			.inputHighlight {
+				height: 72px;
+			}
+		}
+		.redInput,
```
Comment (stanca-pop-lgp) [inline sandstone/custom-skin/src/components/TripleField/TripleField.module.less:41]: Please leave an empty space above.

File: sandstone/custom-skin/src/components/TripleField/TripleField.module.less [hunk 38]
```diff
@@ -1,16 +1,95 @@
...
+		.greenInput,
+		.blueInput {
+			height: 72px;
+			max-width: 129px;
+			margin: 6px;
+			.input {
+				font-size: 27px;
+			}
+			.input,
```
Comment (stanca-pop-lgp) [inline sandstone/custom-skin/src/components/TripleField/TripleField.module.less:36]: Please leave an empty space above.

File: sandstone/custom-skin/src/components/TripleField/TripleField.module.less [hunk 39]
```diff
@@ -1,16 +1,95 @@
...
+		.tripleText {
+			font-size: 27px;
+		}
+		.redInput,
+		.greenInput,
+		.blueInput {
+			height: 72px;
+			max-width: 129px;
+			margin: 6px;
+			.input {
```
Comment (stanca-pop-lgp) [inline sandstone/custom-skin/src/components/TripleField/TripleField.module.less:33]: Please leave an empty space above.

File: sandstone/custom-skin/src/components/TripleField/TripleField.module.less [hunk 40]
```diff
@@ -1,16 +1,95 @@
...
+}
+
+@media only screen and (max-width: 830px) {
+	.tripleField {
+		flex-direction: column;
+		align-items: flex-start;
+		.tripleText {
+			font-size: 27px;
+		}
+		.redInput,
```
Comment (stanca-pop-lgp) [inline sandstone/custom-skin/src/components/TripleField/TripleField.module.less:27]: Please leave an empty space above.

File: sandstone/custom-skin/src/components/TripleField/TripleField.module.less [hunk 41]
```diff
@@ -1,16 +1,95 @@
...
+		max-width: 240px;
+		margin: 0 15px;
+	}
+}
+
+@media only screen and (max-width: 830px) {
+	.tripleField {
+		flex-direction: column;
+		align-items: flex-start;
+		.tripleText {
```
Comment (stanca-pop-lgp) [inline sandstone/custom-skin/src/components/TripleField/TripleField.module.less:24]: Please leave an empty space above.

File: sandstone/custom-skin/src/components/SingleField/SingleField.module.less [hunk 42]
```diff
@@ -1,9 +1,41 @@
...
+
+@media screen and (min-width: 830px) and (max-width: 1670px) {
+	.singleField {
+		.singleInput {
+			height: 96px;
+			max-width: 288px;
+			.input {
+				font-size: 42px;
+			}
+			.input,
```
Comment (stanca-pop-lgp) [inline sandstone/custom-skin/src/components/SingleField/SingleField.module.less:35]: Please leave an empty space above.

File: sandstone/custom-skin/src/components/SingleField/SingleField.module.less [hunk 43]
```diff
@@ -1,9 +1,41 @@
...
+		}
+	}
+}
+
+@media screen and (min-width: 830px) and (max-width: 1670px) {
+	.singleField {
+		.singleInput {
+			height: 96px;
+			max-width: 288px;
+			.input {
```
Comment (stanca-pop-lgp) [inline sandstone/custom-skin/src/components/SingleField/SingleField.module.less:32]: Please leave an empty space above.

File: sandstone/custom-skin/src/components/SingleField/SingleField.module.less [hunk 44]
```diff
@@ -1,9 +1,41 @@
...
+
+@media screen and (max-width: 830px) {
+	.singleField {
+		.singleInput {
+			height: 96px;
+			max-width: 240px;
+			.input {
+				font-size: 33px;
+			}
+			.input,
```
Comment (stanca-pop-lgp) [inline sandstone/custom-skin/src/components/SingleField/SingleField.module.less:19]: Please leave an empty space above.

File: sandstone/custom-skin/src/components/SingleField/SingleField.module.less [hunk 45]
```diff
@@ -1,9 +1,41 @@
...
+		width: 432px;
+	}
+}
+
+@media screen and (max-width: 830px) {
+	.singleField {
+		.singleInput {
+			height: 96px;
+			max-width: 240px;
+			.input {
```
Comment (stanca-pop-lgp) [inline sandstone/custom-skin/src/components/SingleField/SingleField.module.less:16]: Please leave an empty space above.

File: sandstone/custom-skin/src/components/OutputField/OutputField.module.less [hunk 46]
```diff
@@ -1,12 +1,29 @@
...
+		text-transform: uppercase;
+	}
+}
+
+@media only screen and (max-width: 1670px) {
+	.outputField {
+		pre {
+			font-size: 27px;
+		}
+		.copyBtn {
```
Comment (stanca-pop-lgp) [inline sandstone/custom-skin/src/components/OutputField/OutputField.module.less:19]: Please leave an empty space above.

File: sandstone/custom-skin/src/components/ImportSkin/ImportSkin.module.less [hunk 47]
```diff
@@ -1,35 +1,55 @@
...
+			&::before {
+				font-size: 27px;
+				padding: 0 48px;
+			}
+		}
+	}
+
+	@media only screen and (min-width: 870px) and (max-width: 1670px) {
+		font-size: 42px;
+		input[type=file] {
```
Comment (stanca-pop-lgp) [inline sandstone/custom-skin/src/components/ImportSkin/ImportSkin.module.less:49]: Please leave an empty space above.

File: sandstone/custom-skin/src/components/ImportSkin/ImportSkin.module.less [hunk 48]
```diff
@@ -1,35 +1,55 @@
...
+			box-shadow: 0 36px 36px rgb(0 0 0 / 30%);
+			background-color: #e6e6e6;
+			color: #4c5059;
+			cursor: pointer;
+		}
+	}
+
+	@media screen and (min-width: 720px) and (max-width: 870px) {
+		font-size: 33px;
+		input[type=file] {
```
Comment (stanca-pop-lgp) [inline sandstone/custom-skin/src/components/ImportSkin/ImportSkin.module.less:38]: Please leave an empty space above.

File: sandstone/custom-skin/src/components/ImportSkin/ImportSkin.module.less [hunk 49]
```diff
@@ -1,35 +1,55 @@
...
+			color: transparent;
+			&::before {
```
Comment (stanca-pop-lgp) [inline sandstone/custom-skin/src/components/ImportSkin/ImportSkin.module.less:40]: Please leave an empty space above.

File: sandstone/custom-skin/src/components/ImportSkin/ImportSkin.module.less [hunk 50]
```diff
@@ -1,35 +1,55 @@
...
+				font-size: 27px;
+				padding: 0 48px;
+			}
+		}
+	}
+
+	@media only screen and (min-width: 870px) and (max-width: 1670px) {
+		font-size: 42px;
+		input[type=file] {
+			&::before {
```
Comment (stanca-pop-lgp) [inline sandstone/custom-skin/src/components/ImportSkin/ImportSkin.module.less:50]: Please leave an empty space above.

File: sandstone/custom-skin/src/common/styles.module.less [hunk 51]
```diff
@@ -1,34 +1,71 @@
...
+	}
+}
+
+@media only screen and (min-width: 1320px) and (max-width: 1670px) {
+	.colorBlock[type=color] {
+		height: 96px;
+		width: 96px;
+		min-width: 96px;
+	}
+	.inputField {
```
Comment (stanca-pop-lgp) [inline sandstone/custom-skin/src/common/styles.module.less:66]: Please leave an empty space above.

**Review by stanca-pop-lgp — APPROVED** [review]
LGTM

**Pull request merged**